nav2_behavior_tree/include/nav2_behavior_tree/bt_action_server_impl.hpp

```diff
@@ -82,6 +82,9 @@ bool BtActionServer<ActionT>::on_configure()
       "-r",
       std::string("__node:=") +
       std::string(node->get_name()) + "_" + client_node_name + "_rclcpp_node",
+      "-p",
+      "use_sim_time:=" +
+      std::string(node->get_parameter("use_sim_time").as_bool() ? "true" : "false"),
```

Comment:
What if its not set in the yaml + they used different launch files without the set parameter?

Reply:
![image](https://user-images.githubusercontent.com/14879660/186003478-192ccde0-9232-41cc-90c5-c990c2d25cea.png)

in case you're wondering, doesn't apply for any other param

Reply:
Huh. OK, good to know